### DISCUSSION TITLE
Enhanced PDF Export Configuration for Document Info in Polarion Live Documents

### DISCUSSION THREAD

**Discussion by ukrishne** [SchweizerischeBundesbahnen/ch.sbb.polarion.extension.pdf-exporter#85] [Q&A] [answered]
### Description

Introduce a feature that allows users to include additional document information in PDF exports from Polarion Live Documents. This configuration option would enable users to export comprehensive metadata such as Baseline information, Date & Time of export, Applied Filters, and Links info (including the number of outlinks and inlinks with source and destination documents).
Please also add statement that "This document has been generated automatically out of Polarion" so that it is easy to identiify the source of document.

### Motivation

This feature aims to provide a more detailed and informative PDF export that includes critical metadata about the document. By incorporating Baseline details, export Date & Time, applied filters, and link information, users can ensure that the exported PDFs carry all necessary context and traceability information, which is crucial for documentation, auditing, and review processes.

Additional Context:
Example Use Cases:

Baseline Information: Include details such as the Baseline number and description to ensure that the specific version of the document is clearly identified.
Date & Time of Export: Display the exact date and time when the document was exported, providing a clear timestamp for reference.
Applied Filters: Show the filter conditions applied during export to reflect the specific subset of data included in the PDF.
Links Info: Indicate the number of outlinks and inlinks, along with the source and destination documents, to provide a complete picture of document interrelationships.

### Proposed Solution (optional)

User Interface: Add options in the export settings to enable the inclusion of Baseline information, Date & Time of export, applied filters, and links info.
Formatting: Ensure that the additional information is presented clearly and consistently in the exported PDF, possibly in a dedicated section at the beginning or end of the document.
Customization Options: Provide checkboxes or toggles in the export configuration to allow users to select which pieces of information to include in the PDF export.
By implementing this feature, Polarion Live Documents will offer a more robust and informative PDF export capability, catering to the needs of users who require detailed documentation and traceability in their exported files

**Comment by grigoriev** [comment, marked as answer]
Hi @ukrishne ,

have you seen "Header and Footer" options?
We have implemented mechanism of adding headers/footers with required information.

You can use special placeholders for producing document with required info (like Baseline, revision, etc):
<img width="1363" alt="image" src="https://github.com/SchweizerischeBundesbahnen/ch.sbb.polarion.extension.pdf-exporter/assets/4573001/0817bbbb-5e72-451c-988a-2a53bcb1e79c">

Please check these options first.

**Comment by ukrishne** [comment]
ok agreed, can be closed
